lib/ansible/module_utils/urls.py

```diff
@@ -82,10 +82,14 @@
 # Agreement.
 
 try:
```

Comment:
Add a comment to this that we may be able to use six.moves for this once we have ziploader.